scripts/lib/install-node

```diff
@@ -18,7 +18,9 @@ n=${n%/bin/node}
 chown -R root:root "$n"
 chmod -R go-w "$n"
 
-# Install node and npm wrappers to /usr/local/bin
+# Install node wrapper to /usr/local/bin
 cp "$ZULIP_PATH/../../scripts/setup/node-wrapper" /usr/local/bin/node
 sed -i "s|NODE_PATH|$NODE_BIN|" /usr/local/bin/node
-cp "$ZULIP_PATH/../../scripts/setup/npm-wrapper" /usr/local/bin/npm
+
+# Install yarn
+bash "$ZULIP_PATH/../../scripts/lib/third/install-yarn.sh" --version 0.24.6
```

Comment:
I see, the reason was that it was installed twice: first here (where this script was meant just for installing node) and then `setup_node_modules` in `./tools/lib/provision.py`.